src/GridList/GridTile.js

```diff
@@ -182,6 +187,7 @@ class GridTile extends Component {
     const {prepareStyles} = this.context.muiTheme;
     const styles = getStyles(this.props, this.context);
     const mergedRootStyles = Object.assign(styles.root, style);
+    const mergedTitleStyles = Object.assign(styles.title, titleStyle);
```

Comment:
`mergedTitleStyles` variable is useless from a feature point of view.
I think that it would be better to move `Object.assign(styles.title, titleStyle)` to his side effect impact:
```jsx
<div style={prepareStyles(Object.assign(styles.title, titleStyle))}>{title}</div>
```